finder.go

```diff
@@ -70,32 +63,29 @@ func (c Collector) AsCSV() string {
 type DumpType uint8
 
 const (
-	DUMP_TYPE_ANY     DumpType = 0 // any
-	DUMP_TYPE_RIB     DumpType = 1 // rib
-	DUMP_TYPE_UPDATES DumpType = 2 // updates
+	DumpTypeAny     DumpType = 0 // any
+	DumpTypeRib     DumpType = 1 // rib
+	DumpTypeUpdates DumpType = 2 // updates
 )
 
-// TODO: think about how this should work -- just keep it simple! no
-// complex query structures
-//
-// TODO: add Validate method (e.g., From is before Until, IsADumpType, etc.)
+// TODO: think about how this should work -- just keep it simple! no complex query structures
+// TODO: add Validate method (e.g., Start is before End, IsADumpType, etc.)
 type Query struct {
 	// Collectors to search for. All collectors if unset/empty
 	Collectors []Collector
 
 	// Query window start time (inclusive)
-	From time.Time
+	Start time.Time
```

Comment:
Any particular reason for this rename?

Reply:
I want to use the same language as https://bgpstream.caida.org/docs/api/broker to describe the parameters. (The broker uses `{epoch-start},{epoch-end}`).

Reply:
I think we should avoid making this code match exactly match the legacy broker. We want a compatible HTTP API, but I think we should consider it only a part of what this service will do. 

I can imagine that at some point we could release a new version of BGPStream that makes use of a v2 broker.

Basically, let's make this service make the most sense for today rather than making its sole purpose in life to fulfill a 10 year old API contract.

Reply:
I see. That's good to know. I was under the impression that we want the BGPFinder to be completely compatible with the broker.

That said, this opens up several points for discussion. For instance, what about other differences between the broker and BGPFinder? Currently, the project argument is required for BGPFinder’s File subcommand, whereas, for the broker, it is not; if no project is specified, it presumably iterates through all projects.

I think it makes more sense to make the `project` a required field so that we don't run into scaling issues if more projects are added, but would like to hear your thoughts.

Reply:
It's perfectly reasonable to request "all data" from the broker. So project needn't be required.

Also let's not forget that the BGPFinder _CLI_ can have a different set of requirements to the broker compat API to a possible future BGPFinder-native API. We want the CLI to be ergonomic for a user doing some exploration.